### DISCUSSION TITLE
Cannot control Reframwork menu settings through game screen

### DISCUSSION THREAD

**Discussion by breemzoner** [praydog/REFramework#373] [General Questions]
I cannot control the Re framework menu screen when I launch games - all my input goes directly to the game itself and not the menu on my monitor screen. Any help?  I want to enable passthrough

**Comment by praydog** [comment]
Use the VR menu instead. https://cursey.github.io/reframework-book/troubleshooting/VR-Troubleshooting.html#opening-the-in-game-menu-with-motion-controllers-openvr-only-right-now

**Reply by breemzoner** [reply]
I can't because on my pimax 8k headset the game is not being rendered on the headset itself which is why I have to manually alter it through the menu on the in game monitor screen.

**Reply by praydog** [reply]
> I can't because on my pimax 8k headset the game is not being rendered on the headset itself which is why I have to manually alter it through the menu on the in game monitor screen.

If you are unable to use the method I linked to, you can modify configuration options in your `re2_fw_config.txt`
